projects/js-packages/critical-css-gen/package.json

```
@@ -0,0 +1,91 @@
+{
+	"private": true,
+	"type": "module",
+	"name": "@automattic/jetpack-critical-css-gen",
+	"version": "0.1.0-alpha",
+	"description": "A flexible Critical CSS Generator that supports multiple URLs and viewports, with both server-side and client-side generation capabilities.",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/critical-css-gen/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Critical Css Gen"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/critical-css-gen"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build:browser": "rollup -c",
+		"build:node": "tsc --project tsconfig.node.json",
+		"build": "npm run clean && npm run build:browser && npm run build:node",
+		"clean": "rm -rf lib/ && rm -rf dist/",
+		"lint": "prettier --check src/**/* && eslint . && echo '✔ eslint and prettier ran successfully.'",
+		"lint:fix": "prettier --write src/**/*.ts && eslint --fix . && echo '✔ eslint and prettier ran successfully.'",
+		"test": "npm run build && NODE_ENV=test NODE_PATH=./node_modules jest --forceExit --config=tests/config/jest.config.js"
+	},
+	"main": "./lib/back-end.js",
+	"browser": "./dist/bundle.js",
+	"devDependencies": {
+		"@babel/core": "7.24.7",
+		"@babel/preset-env": "7.24.7",
+		"@babel/preset-typescript": "7.24.7",
+		"@rollup/plugin-commonjs": "26.0.1",
+		"@rollup/plugin-json": "4.1.0",
+		"@rollup/plugin-node-resolve": "13.3.0",
+		"@rollup/plugin-terser": "0.4.3",
+		"@rollup/plugin-typescript": "8.3.3",
+		"@types/clean-css": "4.2.5",
+		"@types/css-tree": "2.0.1",
+		"@types/node": "^20.4.2",
+		"@typescript-eslint/eslint-plugin": "6.21.0",
+		"@typescript-eslint/parser": "6.21.0",
+		"eslint": "8.57.0",
+		"express": "4.18.1",
+		"https-browserify": "1.0.0",
+		"jest": "29.7.0",
+		"jest-environment-puppeteer": "10.0.1",
+		"node-fetch": "3.3.2",
+		"nodemon": "3.1.2",
+		"os-browserify": "0.3.0",
+		"path-browserify": "1.0.1",
+		"playwright-core": "1.33.0",
+		"prettier": "npm:wp-prettier@2.0.5",
```

To match the rest of the monorepo,
```suggestion
		"prettier": "npm:wp-prettier@3.0.3",
```

---

Updated in 16149266c4915e129d54e79cd92c49aa40294695.